### ISSUE TITLE
tiny bug

### ISSUE THREAD

**Issue by vanja-san** [lantica/line_sticker_converter#8]
Script skips some stickers

**Comment by lantica** [comment]
Hi @vanja-san , could you provide an example sticker pack id for me to test.
And I think this repo only support static sticker at this moment.

**Comment by vanja-san** [comment]
@lantica 16800998
or 18361661 <-- there's a lot not downloading (40 stickers, downloaded 27)

**Comment by lantica** [comment]
Thanks you for finding this bug.
I identified where the bug is but I don't know why this happen.
It should be fixed in 1.0.3.
I change it back to extract the sticker to the temp directory first and this seems solved the issue.


The problem happened with the if else check, despite it's returning true for the regex test, it will still enter the else clause.
I have completely no idea why would this happen.

```
    .pipe(
        new Transform({
            objectMode: true,
            transform: (entry, e, cb) => {
                if (nameRegex.test(entry.path)) {
                    entry
                        .pipe(stickerResizerFactory())
                        .pipe(fs.createWriteStream(path.resolve(outputPath, `${path.parse(entry.path).name}_512.png`)))
                        .on("finish", cb);
                } else {
                    entry.autodrain();
                    cb();
                }
            },
        })
    )
```

**Comment by vanja-san** [comment]
@lantica Now I tried, everything is downloaded completely. Thanks for the quick fix!
